### ISSUE TITLE
The function placed next to class definition cannot be found

### ISSUE THREAD

**Issue by mwu-tow** [luna/luna#217]
Please consider the following code:
```
class Foo:
    foo :: Int

def sayHello: print "Hello!"
def main: sayHello
```

It fails to compile with the following error: 
```- Can't find function sayHello```
I'd expect it to compile successfully.

The issue goes away if the `dummy` method is added to the Client class:
```
class Foo:
    foo :: Int
    def dummy: None

def sayHello: print "Hello!"
def main: sayHello
```

**Comment by wdanilo** [comment]
Oh, this one is funny. I know about this bug and the good news is that it is already fixed on our Luna 2.0 branch. Sorry that it isn’t fixed here as well but we’ve found it while working on parser working on new core (with many changes). I’ll leave it open. Sorry Michal for inconvinence!

**Comment by mikusp** [comment]
This issue is resolved
